src/mame/includes/tsconf.h

```diff
@@ -10,33 +10,32 @@
 
 #pragma once
 
-#include "spectrum.h"
-
 #include "machine/beta.h"
-
 #include "machine/glukrs.h"
 #include "machine/pckeybrd.h"
 #include "machine/spi_sdcard.h"
 #include "machine/tsconfdma.h"
-
+#include "spectrum.h"
 #include "tilemap.h"
 
+#define TSCONF_SCREEN_HBLANK 88
+#define TSCONF_SCREEN_VBLANK 32
```

Comment:
It’s better to avoid macros when a constexpr will do, and better to keep stuff out of the global namespace if it doesn’t need to be there.  Since these are used in a file static, could you just make them `constexpr` here?  The compiler doesn’t need to instantiate storage for them if nothing tries to take their addresses.

Reply:
How about this?